.devcontainer/docker-compose.extend.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+
+  watch-static-files:
+    security_opt:
+      - seccomp:unconfined
+
+  backend:
+    security_opt:
+      - seccomp:unconfined
+    volumes:
+      - ~/.ssh:/root/.ssh-localhost:ro
+    command: /bin/sh -c "while sleep 1000; do :; done"
```

Comment:
???

Reply:
[According to the docs](https://code.visualstudio.com/docs/remote/containers#_using-an-image-or-dockerfile):

By default, when VS Code starts a container, it will override the container's default command to be `/bin/sh -c "while sleep 1000; do :; done"`. This is done because the container will stop if the default command fails or exits

I pulled this out of a sample docker-compose override in the advanced-containers setup guide.